_posts/2021-08-02-changelog-152.markdown

```diff
@@ -0,0 +1,206 @@
+
+    {% include gif-img.html path="/assets/posts/changelog-152/convert-function-to-closure" w="600px" %}
+
+  * [#6813] `Convert closure to function` to convert a lambda expression to a local named function (by [@nschoellhorn])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/convert-closure-to-function" w="600px" %}
+
+  * [#6414] `Replace successive '_' with '..'` to replace redundant wildcards with a single `..` (by [@zeroeightysix])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/replace-wildcards" w="600px" %}
+
+  * [#7559] `Simplify dependency specification` to ...TODO inline tables in `Cargo.toml` dependencies
+
+    {% include gif-img.html path="/assets/posts/changelog-152/simplify-dependency-specification" w="600px" %}
+
+* Completion improvements:
+
+  * [#7487], [#7551], [#7536] Add completion for lambdas in expression positions that expect a function (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/lambda-completion" w="600px" %}
+
+  * [#7481] Add completion for `pub(crate)` and `pub()` visibility modifers (by [@Kobzol])
+
+  * [#6410] Add completion for layouts in `#[repr()]` attributes (by [@zeroeightysix])
+
+  * [#7534], [#7571] Improve completion of items inside impl trait (by [@Kobzol])
+
+    {% include gif-img.html path="/assets/posts/changelog-152/completion-member" w="600px" %}
+
+* [#7532] Add quick fix to implement missing supertrait(s) for a trait impl block (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-152/implement-supertraits" w="600px" %}
+
+* [#7492] Add a quick fix to remove an unused import (by [@Kobzol])
+
+* [#7479] TY&RES: support const arguments that looks like type arguments
+
+* [#7519] Add postfix template named `.wrap` that wraps an existing type with another type (by [@Kobzol])
+
+* [#7499] added a new filter to the Structure View panel, which toggles whether macro-generated items
+  should be included in the Structure View (by [@afetisov])
+
+* [#7495] The import optimizer will now remove unused imports (by [@Kobzol])
+
+* [#7321] `show macro expansion` actions and intentions now show a reason of the expansion failure
+
+* [#7308] Provide an experimental feature to add `--nocapture` flag to `cargo test` command
+
+* [#7228] Make links that look like Rust identifiers or paths (e.g., A, A::B, A::B::C) work correctly in
+  quick documentation by prepending `psi_element://` to them. It fixes #7209
+  Now clickable links inside stdlib docs should work properly (by [@shocoman])
+
+* [#6003] Allow inlining functions (using `Inline Function` refactoring) with diverging exit points
+  (like `panic!()`) (by [@SaarYogev])
+
+* [#7563] Add option to enable unused import inspection in doctests
+
+## Performance Improvements
+
+* [#7576] Slightly optimize `Find usage` for private items inside files with child modules
+
+## Fixes
+
+* [#7521], [#7503] TODO-rename Support type aliases for all types (by [@Kobzol])
```

Comment:
Should we mention these changes? Are they visible to users?

Reply:
Yes, for example, in inlay type hints